profiles/common/modules/custom/tmgmt_dgt_html/tmgmt_dgt_html.module

```diff
@@ -41,17 +41,24 @@ function template_preprocess_tmgmt_dgt_html_template(&$variables) {
 
   foreach (array_keys($variables['items']) as $tjid) {
     foreach (array_keys($variables['items'][$tjid]) as $data_id) {
-      $escape_for_html = TRUE;
+      $is_html = FALSE;
 
       if (isset($variables['items'][$tjid][$data_id]['#format'])) {
         $format = $variables['items'][$tjid][$data_id]['#format'];
 
-        if (in_array($format, $html_formats)) {
-          $escape_for_html = FALSE;
-        }
+        $is_html = in_array($format, $html_formats);
       }
 
-      if ($escape_for_html) {
+      if ($is_html) {
+        // Ensure line break tags are always properly closed in order to be
+        // XHTML compliant.
+        $variables['items'][$tjid][$data_id]['#text'] = str_replace(
```

Comment:
I know that any Full HTML format does not propose the corresponding button but should we cover the "hr" tag treatment too?